x/btccheckpoint/README.md

```diff
@@ -0,0 +1,222 @@
+# BTCCheckpoint
+
+The BTC Checkpoint module is responsible for receiving and managing the checkpoints of Babylon's state on the Bitcoin blockchain. This includes:
+- handling requests for submitting raw checkpoints
+- processing Bitcoin SPV proofs for submitted checkpoints
+- managing the lifecycle of checkpoints (SUBMITTED, CONFIRMED, FINALIZED, SEALED)
```

Comment:
I would re-arrange it into:
```
 (SEALED, SUBMITTED, CONFIRMED, FINALIZED,)
```
as this is order of events i.e

- checkpoint is sealed on Babylon chain
- checkpoint is submitted to BTC
- checkpoint is confirmed on BTC
- checkpoint is finalized on BTC